.github/workflows/CI.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  macos:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build
+      run: swift build -v
+    - name: Tests
+      run: swift test -v
+  # linux:
+  #   runs-on: ubuntu-latest
+  #   container:
+  #     image: swift:latest
+  #   steps:
+  #   - uses: actions/checkout@v3
+  #   - name: Build
+  #     run: swift build -v
+  #   - name: Tests
+  #     run: swift test -v
```

Comment:
Builds on linux broken now, but tried setting it up for now. 
Will fix in another PR (need to address `Combine`)